kong/db/strategies/postgres/connector.lua

```diff
@@ -634,15 +648,37 @@ function _mt:schema_migrations()
 end
 
 
-function _mt:schema_bootstrap(kong_config, default_locks_ttl)
+function _mt:schema_bootstrap(_, default_locks_ttl)
```

Comment:
There is no reason for this change since our linter ignores unused arguments. However, this changes makes it harder for future developers (and selves) to remember what this argument is. It is less explicit and thus less self-documented.

Reply:
Reverted that change.